iOS_SDK/OneSignalDevApp/OneSignalDevApp/SwiftTest.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SwiftTest.swift
+//  OneSignalExample
+//
+//  Created by Elliot Mawby on 12/12/22.
+//  Copyright © 2022 OneSignal. All rights reserved.
```

Comment:
Fix copyright header to match our standard.

Reply:
done!